desktop/src/main.rs

```diff
@@ -216,6 +217,22 @@ fn load_movie(url: &Url, opt: &Opt) -> Result<SwfMovie, Error> {
     Ok(movie)
 }
 
+fn get_max_screen_size(monitors: impl IntoIterator<Item = MonitorHandle>) -> (u32, u32) {
+    let mut max_screen_size = (0, 0);
+
+    for monitor in monitors {
+        let size = MonitorHandle::size(&monitor);
+        max_screen_size.0 += size.width;
+        max_screen_size.1 += size.height;
+    }
+
+    if max_screen_size.0 <= 64 || max_screen_size.1 <= 32 {
+        return (32767, 32767);
```

Comment:
This is `(i16::MAX, i16::MAX)` right? Can you change it to those constants, just so we know it's max and not a value with any special significance?

Reply:
Also why this case is needed?

Reply:
Just a sanity check, in case winit couldn't detect any monitors we don't want it to return a max size of 0,0. But if the monitor detection always succeeds we wouldn't need it.